creator-node/src/services/prometheusMonitoring/prometheus.constants.js

```diff
@@ -21,12 +41,43 @@ const MetricTypes = Object.freeze({
 let MetricNames = {
   SYNC_QUEUE_JOBS_TOTAL_GAUGE: 'sync_queue_jobs_total',
   ROUTE_POST_TRACKS_DURATION_SECONDS_HISTOGRAM:
-    'route_post_tracks_duration_seconds'
+    'route_post_tracks_duration_seconds',
+  ISSUE_SYNC_SECONDS_WAITING_FOR_COMPLETION_HISTOGRAM:
+    'issue_sync_seconds_waiting_for_completion_histogram'
```

Comment:
I'll change to have duration followed by units. for the type -- other metrics include it, and the readme explicitly says to include it ([link](https://github.com/AudiusProject/audius-protocol/blob/master/creator-node/src/services/prometheusMonitoring/README.md#create-metric-in-prometheusconstantsjs))

> Key should include a suffix for metric type (either _GAUGE or _HISTOGRAM)

Reply:
oh are you just talking about for the actual name? keep the type everywhere else except the string value?

Reply:
yep exactly
prometheus route exports a lot of data, so putting metric type in the string value itself is redundant since we already know what type it is
e.g. https://creatornode6.staging.audius.co/prometheus_metrics (cmd+f histogram)

Reply:
sorry docs not clear there, can you update